### DISCUSSION TITLE
Is there a way to have both Enter keys (normal and num pad) have the same function, please?

### DISCUSSION THREAD

**Discussion by nitrostudios** [mikekazakov/nimble-commander#456] [Ideas]
Is there a way to have both Enter keys (normal and num pad) have the same function, please? Now I can't open folders and open files while using the Enter key on num pad in NC, while I can do the same in other applications. It would be great if it can be implemented. Thank you very much!

**Comment by djdakta** [comment]
You can eventually change the "Hotkey" in NC's settings and this is the one you need/want to change:

![NC_Hotkeys_0001](https://github.com/user-attachments/assets/08e3e2af-f6df-4bac-b847-2a46cfdd308d)

**Reply by nitrostudios** [reply]
Hello, unfortunately, that does not work, since it's completely different command for selecting the file and moving cursor down. What I'm talking about is ability to use the num pad Enter the same way as normal Enter. Now it's not possible.

**Comment by mikekazakov** [comment]
At the moment, NC allows only one hotkey to be assigned to an action.
There are a few rare exceptions, but I don't think it's a good idea to expand them on an ad-hoc basis.
Return and Enter are two distinct keys, so this scenario (i.e., binding more than a single hotkey) is not supported.

I've been wanting to lift this restriction for a long time. This means adding a generic mechanism to support secondary hotkeys - plural - that is not limited to just one or two hotkeys per action.
Implementing this feature will likely be one of the focus areas for v1.8.

**Comment by mikekazakov** [comment]
This is already supported in v1.8, so this thread can be closed.

**Reply by nitrostudios** [reply]
Thank you very much, just updated and it works great:) Appreciate the support, Mike:)
